records_versioning/services.py

```
@@ -0,0 +1,154 @@
+            version=next_ver,
+            prev_version=None if not last else last.version,
+            snapshot=curr,
+            diff=diff,
+            meta={"reason": reason},
+            changed_by=user,
+        )
+    finally:
+        _thread_state.in_version = False
+
+@transaction.atomic
+def revert_to_version(resource_type: str, resource_id, target_version: int, user, reason="revert"):
```

Ruff (0.12.2)

Missing return type annotation for public function `revert_to_version`

(ANN201)